xbmc/Application.cpp

```diff
@@ -1692,8 +1692,8 @@ bool CApplication::OnAction(const CAction &action)
     }
     else if (hdrStatus == HDR_STATUS::HDR_ON)
     {
-      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info, g_localizeStrings.Get(34222),
-                                            g_localizeStrings.Get(34223));
+      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info, g_localizeStrings.Get(34220),
+                                            g_localizeStrings.Get(34222));
     }
```

Comment:
What about the other HDR_STATUS values? Shouldn't we at least write a warning to the log?

Reply:
It is not needed here. Only the ON / OFF states are used to display the toast. If e.g. system does not support HDR the F11 key is ignored (it's not an error).

Reply:
What, if this gets implemented for other OS's than Windows? I think logging a warning would not harm here.

Reply:
Functionality, logging, etc. should be implemented on windowing override method on each operating system

Reply:
By default on others OS returns `HDR_UNSUPPORTED`
https://github.com/xbmc/xbmc/blob/97f660dbb248d984d65bafd6745d7ffd71022cb2/xbmc/windowing/WinSystem.h#L165

Anyway "HDR toggle" probably is a Windows only thing because has it's own HDR switch in display control panel. Android/Linux does not need it.

Reply:
> Functionality, logging, etc. should be implemented on windowing override method on each operating system

 Fair enough.